Gemfile

```diff
@@ -13,6 +13,9 @@ gem 'devise', '~> 4.7', '>= 4.7.2'
 gem 'devise_token_auth', '~> 1.2', git: 'https://github.com/lynndylanhurley/devise_token_auth'
 gem 'draper', '~> 4.0', '>= 4.0.1'
 gem 'exception_hunter', github: 'rootstrap/exception_hunter'
+gem 'flipper'
+gem 'flipper-active_record'
+gem 'flipper-ui'
```

Comment:
please add versions with `~>` to prevent undesired upgrades (when upgrading some other gem).

https://bundler.io/guides/rationale.html
https://thoughtbot.com/blog/a-healthy-bundle
http://web.archive.org/web/20160413161331/blog.zenspider.com/blog/2008/10/rubygems-how-to-preventing-catastrophe.html